src/ducks/recurrence/config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "amountShouldBeMoreThan": {
+    "active": true,
+    "options": 5
+  },
+  "bundleSizeShouldBeMoreThan": {
+    "active": true,
+    "options": 2
+  },
+  "categoryShouldBeSet": {
+    "active": true
+  },
+  "deltaMeanInferiorTo": {
+    "active": true,
+    "options": 90
+  },
+  "deltaMeanSuperiorTo": {
+    "active": true,
+    "options": 7
+  },
+  "madInferiorTo": {
+    "active": true,
+    "options": 5
+  },
+  "mergeBundles": {
+    "active": true
+  },
+  "sigmaInferiorTo": {
+    "active": false,
```

Comment:
Why is it false?

Reply:
Because we use the median standard deviation instead of the normal standard deviation.